RELEASE.md

```diff
@@ -1,16 +1,20 @@
 # Release Management
 
 Releases of the DSS are based on git tags in the format `v[0-9]+\.[0-9]+\.[0-9]+`.
-When either an executable or image is built from a `git` checkout of the source, the latest tag up to the current commit satisfying the aforementioned format is used as the current version. If no such tags exists, the build system defaults to v0.0.0.
+When either an executable or image is built from a `git` checkout of the source, the [reference](https://git-scm.com/docs/git-describe#:~:text=If%20the%20tag%20points%20to%20the%20commit%2C%20then%20only%20the%20tag%20is%20shown.%20Otherwise%2C%20it%20suffixes%20the%20tag%20name%20with%20the%20number%20of%20additional%20commits%20on%20top%20of%20the%20tagged%20object%20and%20the%20abbreviated%20object%20name%20of%20the%20most%20recent%20commit.)
+as returned by `git describe` is used:
+> If the tag points to the commit, then only the tag is shown. Otherwise, it suffixes the tag name with the number of additional commits on top of the tagged object and the abbreviated object name of the most recent commit.
+
+If no such tags exists, the build system defaults to v0.0.0.
 
 With that, releasing a DSS version and producing release artifacts requires the following steps:
-  * Create a tag `vX.Y.Z` on master and push it to the remote using `make release MAJOR=X MINOR=Y PATCH=Z`
-  * Optionally, build the main docker image, tag it with `vX.Y.Z` and push it out to an image registry of your choice. The official upstream image for a given release is made available at https://hub.docker.com/repository/docker/interuss/dss.
+* Create a tag `vX.Y.Z` on master and push it to the remote using `make release MAJOR=X MINOR=Y PATCH=Z`
```

Comment:
It seems like we only want people to follow this workflow on the `interuss` fork (see top comment).  If that's true, we should specify that ("With that, releasing a DSS version in the canonical interuss fork requires the following steps").  If that's not true (and people can use this release workflow on their own fork as well to accomplish some tasks), we should elaborate on how that will/should work.

Reply:
Please review the new version.